lib/oelite/oven.py

```diff
@@ -26,7 +26,13 @@
 import time
 
 class OEliteOven:
-    def __init__(self, baker, capacity):
+    def __init__(self, baker, capacity=None):
+        if capacity is None:
+            pmake = baker.config.get("PARALLEL_MAKE")
+            if pmake is None or pmake == "":
+                capacity = 3
+            else:
+                capacity = int(pmake.replace("-j", "")) + 2
```

Comment:
Well, as I said, there's room for tuning. But I'm really not a fan of adding a configurable just for the sake of it - how would a user know what a sensible value is?